### PR TITLE
feat: [#1123] members form to business formation

### PR DESCRIPTION
**Pull request by mikewolfd**

### CODE REVIEW

File: web/test/factories.ts [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import { states } from "@/components/tasks/business-formation/StateDropdown";
```
Comment (aloverso) [inline web/test/factories.ts:1]: To once again make my point about moving this `states` export - it is very strange to see a component being important into a test-generator file. And weird to see .tsx being imported into .ts

**Comment by aloverso** [comment]
As an overall point, if possible, I think it'd be ideal to be able to have a refactoring (pre-factoring) commit (the switch to using the `Generic` field components) separate from the new feature of the members work. They're independent bodies of work and could be independently merged/pushed/modified

File: web/src/components/tasks/business-formation/MembersModal.tsx [hunk 2]
```diff
@@ -0,0 +1,264 @@
...
+      });
+      setMemberErrorMap(createMemberErrorMap(false));
+    }
+  };
+
+  const handleClose = () => {
+    props.handleClose();
+    reset();
+  };
+  const onValidation = (invalid: boolean, fieldName: string) => {
```
[the remark this answers is not in the document]
Comment (mikewolfd) [reply web/src/components/tasks/business-formation/MembersModal.tsx:83]: np, could we make this an eslint thing?

File: web/src/components/tasks/business-formation/MembersModal.tsx [hunk 3]
```diff
@@ -0,0 +1,264 @@
...
+  };
+
+  useEffect(() => {
+    if (props.index !== undefined) {
+      setMemberData({ ...state.formationFormData.members[props.index] });
+      setMemberErrorMap(createMemberErrorMap(false));
+    } else {
+      reset();
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```
[the remark this answers is not in the document]
Comment (mikewolfd) [reply web/src/components/tasks/business-formation/MembersModal.tsx:60]: it wanted me to add the functions as deps, which was causing shitty/weird behavior

**Pull request merged**